### PR TITLE
DON-307 make tip input to tbg smaller and less instrusive

### PR DESCRIPTION
**Pull request by tbg-jasonfung**
- DON-307 make tip input to tbg smaller and less instrusive

### CODE REVIEW

File: src/app/donation-start/donation-start.component.scss [hunk 1]
```diff
@@ -100,6 +100,18 @@
   font-size: 22px;
 }
 
+.c-your-donation__form-field-to-tbg {
```
Comment (NoelLH) [inline src/app/donation-start/donation-start.component.scss:103]: @tbg-jasonfung the QA feedback was around making these consistent with each other, so couldn't they just share a class?

Comment (tbg-jasonfung) [reply src/app/donation-start/donation-start.component.scss:103]: @NoelLH think the change is to make the tip input field smaller as outlined by Babsy

Comment (tbg-jasonfung) [reply src/app/donation-start/donation-start.component.scss:103]: @NoelLH sorry I just realised I tagged the wrong ticket, keep doing that recently...see this ticket: https://thebiggive.atlassian.net/browse/DON-307

Comment (NoelLH) [reply src/app/donation-start/donation-start.component.scss:103]: Hm, OK. This seems to be the opposite of the suggestion on DON-308 but I agree it makes sense for it to be smaller.

**Review by NoelLH — APPROVED** [review]

**Pull request merged**